DeusEx/Classes/DXRKeys.uc

```diff
@@ -113,14 +289,39 @@ function MoveNanoKeys4()
             }
             i++;
         }
+        /*if(i==slot) continue;
+        foreach AllActors(class'Containers', c)
```

Comment:
swapping keys with containers gives a lot more options, need to add an option to the menu to enable this, it's maybe risky?